bert_mlm_example/bert_model.py

```
@@ -0,0 +1,35 @@
+
+    def forward(self, input_ids, attention_mask=None):
+        B, L = input_ids.shape
+        if L > self.max_length:
+            raise ValueError(f"sequence length {L} exceeds model max_length {self.max_length}")
+        pos_ids = torch.arange(L, device=input_ids.device).unsqueeze(0).expand(B, L)
+        x = self.tok_emb(input_ids) + self.pos_emb(pos_ids)
+        if attention_mask is not None:
+            # Transformer expects mask with True for positions to skip
+            attn_mask = attention_mask == 0
+        else:
+            attn_mask = None
```

Ruff (0.11.9)

Use ternary operator `attn_mask = attention_mask == 0 if attention_mask is not None else None` instead of `if`-`else`-block

(SIM108)